sw/airborne/modules/gps/gps_ubx.h

```diff
@@ -65,6 +69,7 @@ struct GpsUbx {
   uint8_t send_ck_a, send_ck_b;
   uint8_t error_cnt;
   uint8_t error_last;
+  uint16_t reset;
```

Comment:
would it be interesting to make a setting to ask a reset from the ground station ?

Reply:
This `uint16_t` could also just be an `uint8_t`?